interfaces/PA-App/src/app/services/update.service.ts

```diff
@@ -102,4 +109,20 @@ export class UpdateService {
       itemString
     );
   }
+
+  createUpdateToast(messageKey: string) {
+    this.translate.get(messageKey).subscribe((message: string) => {
```

Comment:
You could use `this.translate.instant(messageKey)` (I think) so you don't need the whole `.subscribe`-callback thing...

But I'm not 100% sure.

Reply:
![image](https://user-images.githubusercontent.com/34537157/63516842-167c0d80-c4ee-11e9-8af7-bba67447ee84.png)
Yes i tried that first :(. But when i did that i only got the message key back e.g. 'TOAST.conclusion' instead of the value. I figured it was because of the reason in the image: https://stackoverflow.com/questions/46216185/ngx-translate-instant-returns-key-instead-of-value

Reply:
But i will recheck

Reply:
It's true


>     const message = this.translate.instant(messageKey);
    const closeButtonText = this.translate.instant('TOAST.close');
    this.toastController.create({
      message,
      animated: true,
      showCloseButton: true,
      closeButtonText,
      cssClass: 'update-toast',
      position: 'bottom'
    }).then((obj) => {
      obj.present();
    });

results in: 

![image](https://user-images.githubusercontent.com/34537157/63517213-d701f100-c4ee-11e9-9345-eb85024420c3.png)